src/Spotify.js

```diff
@@ -0,0 +1,37 @@
+import { URL } from 'url';
+
+const includesSomeOfArray = (string, array) =>
+  array.some(item => string.startsWith(item));
+
+export const shouldTransform = url => {
+  const { host, pathname } = new URL(url);
+
+  return (
+    host === 'open.spotify.com' &&
+    includesSomeOfArray(pathname, [
+      '/album/',
+      '/artist/',
+      '/episode/',
+      '/show/',
+      '/track/',
+      '/playlist/',
+    ])
+  );
+};
+
+export const getSpotifyIFrameSrc = urlString => {
+  const { pathname } = new URL(urlString);
+  const type = pathname.split('/')[1].toLowerCase();
+
+  if (['episode', 'show'].includes(type)) {
```

Comment:
Maybe extract the array into a `podcastTypes` variable, so it's more clear why these 2 are special? 🤔

```suggestion
  const podcastTypes = ['episode', 'show'];
  if (podcastTypes.includes(type)) {
```

Reply:
Sure thing